### PR TITLE
Addition of relevant documentation about adding documentation

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
## Description

Fixes pyrsia/pyrsia#655

This PR does solves clarity about adding documentation by adding relevant docs and you can also test the changes by reading through them.

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

### CODE REVIEW

**Review by fishseabowl — APPROVED** [review]

File: docs/get_involved/documentation.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Pyrsia Contributors
```
Comment (prince-chrismc) [inline docs/get_involved/documentation.md:1]: Just to be consistent with the other naming for similar files

```suggestion
# Documentation Contribution Workflow
```

**Review by prince-chrismc — COMMENTED** [review]
In #655 there's more items that need to be address that I dont see covered in this PR

> - Markdown Level 1 heading will be page titles (and more) on the website
> - readme are used to give the "folders" a nice name on the website

As @betarelease released mentioned in the retro it's a great idea to explain what documentation can be found in each sub section

I think these items are important part before closing the issue

File: docs/get_involved/doc_workflow.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[front matter for position](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#sidebar_position)
+the folder on the website. Make sure to pay attention to the other folder numbers.
+
+All documents should have a `.md` file extention. If you need more customization you can use `.mdx`, see 
+[here](https://docusaurus.io/docs/markdown-features/react) for more imformation. All files need a level one heading to provide the
+page with a good title.
+
+## Previewing Changes
+
+Currently it is only possible to preview you changes locally, you should be following the [contributing guidelines](https://pyrsia.io/docs/get_involved/contributing/#dev-flow) and have worked the repository. To preview the changes:
```
Comment (erwin1) [inline docs/get_involved/doc_workflow.md:26]:
```suggestion
Currently it is only possible to preview your changes locally, you should be following the [contributing guidelines](https://pyrsia.io/docs/get_involved/contributing/#dev-flow) and have worked the repository. To preview the changes:

```

Comment (erwin1) [reply docs/get_involved/doc_workflow.md:26]: I'm not sure what 'have worked the repository' means? but that could be just me :)

File: docs/get_involved/doc_workflow.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Clone the [website's repository](https://github.com/pyrsia/pyrsia.github.io) or your own fork.
```
Comment (erwin1) [inline docs/get_involved/doc_workflow.md:28]: if you need a fork to preview, it's better to state that in this step already? 
```suggestion
1. Fork the [website's repository](https://github.com/pyrsia/pyrsia.github.io) and clone your fork.

```

File: docs/get_involved/doc_workflow.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+2. Modify [this line](https://github.com/pyrsia/pyrsia.github.io/blob/main/package.json#L6)
+   - Change `pyrsia/pyrsia` to your fork, (e.g `octocat/pyrsia`)
+   - Optionally, you can change the branch by replacing `main` with `your-branch-name`
+   - You can [check this example](https://github.com/pyrsia/pyrsia.github.io/pull/66/commits/c317f9dab8f6bcde5f8588ca75858db72241930d)
+4. Follow instructions described [here](https://github.com/pyrsia/pyrsia.github.io#website) for "local development"
+
+If you make changes to you fork of the `pyrsia/pyrsia` repository, you can restart the local server to update documentation.
```
Comment (erwin1) [inline docs/get_involved/doc_workflow.md:35]:
```suggestion
If you make changes to your fork of the `pyrsia/pyrsia` repository, you can restart the local server to update documentation.

```

File: docs/get_involved/doc_workflow.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+the folder on the website. Make sure to pay attention to the other folder numbers.
+
+All documents should have a `.md` file extention. If you need more customization you can use `.mdx`, see 
+[here](https://docusaurus.io/docs/markdown-features/react) for more imformation. All files need a level one heading to provide the
+page with a good title.
+
+## Previewing Changes
+
+Currently it is only possible to preview your changes locally, you should be following the [contributing guidelines](https://pyrsia.io/docs/get_involved/contributing/#dev-flow) and have worked the repository. To preview the changes:
+
```
Comment (prince-chrismc) [inline docs/get_involved/doc_workflow.md:27]:
```suggestion
Currently it is only possible to preview your changes locally, you should be following the [contributing guidelines](https://pyrsia.io/docs/get_involved/contributing/#dev-flow) and have forked the repository. To preview the changes:

```

**Review by erwin1 — APPROVED** [review]

**Review by fishseabowl — APPROVED** [review]
LGTM

**Pull request merged**